INSTALL.md

```diff
@@ -206,9 +206,7 @@ Set your `QIITA_CONFIG_FP` environment variable to point to that file (into `.ba
   source activate qiita
 ```
 
-Update paths in the newly copied configuration file to match your settings, e.g. replace /home/travis/ with your user home directory.
-
-If you are working on WSL, you will need to start the redis server with the following command before making a test environment:
+If you are working on Mac OS X or WSL, you will need to start the redis server with the following command before making a test environment:
```

Comment:
Install on mac (not just on WSL) fails if you don't run this.  Ask me how I know :)